src/statics/flag-icons/flags/1x1/ae.svg

```diff
@@ -0,0 +1,6 @@
+<svg xmlns="http://www.w3.org/2000/svg" id="flag-icon-css-ae" viewBox="0 0 512 512">
```

Comment:
why are you adding all these flags here? We already had a component with flags bundled as CSS.

Reply:
The unused flags will be removed as updates go on :smile:

Reply:
The already bundled vue-country flag when used deforms the ui.